stm32-modules/heater-shaker/firmware/system/system_policy.hpp

```diff
@@ -1,6 +1,50 @@
 #pragma once
 
+#include <array>
+#include <iterator>
+#include <ranges>
+
+#include "heater-shaker/errors.hpp"
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wvolatile"
+#include "system_hardware.h"
```

Comment:
Rather than doing this, let's but the implementation of these methods in the `.cpp`